src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -2489,20 +2490,18 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     ====================================================================================================
   -->
   <Target Name="GenerateBindingRedirectsUpdateAppConfig"
-    AfterTargets="GenerateBindingRedirects"
-    Condition="'$(AutoGenerateBindingRedirects)' == 'true' and '$(GenerateBindingRedirectsOutputType)' == 'true' and Exists('$(_GenerateBindingRedirectsIntermediateAppConfig)')">
+    Condition="'$(AutoGenerateBindingRedirects)' == 'true' and '$(GenerateBindingRedirectsOutputType)' == 'true'">
 
-    <PropertyGroup>
+    <PropertyGroup Condition="Exists('$(_GenerateBindingRedirectsIntermediateAppConfig)')">
```

Comment:
I'm a little concerned by creeping disk pings in the common case for things like this. Is there any way we can make it go away?

Reply:
Merged into a property.